README.md

```diff
@@ -1,10 +1,5 @@
 # DiffEqFlux.jl
 
-[![Join the chat at https://gitter.im/JuliaDiffEq/Lobby](https://badges.gitter.im/JuliaDiffEq/Lobby.svg)](https://gitter.im/JuliaDiffEq/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
CI badges and such should not be removed.

Reply:
I'll revert.

Reply:
But I changed the gitlab repo to juliadiffeq. SciML doesn't seem to exist.

Reply:
oh yes, the Gitter is still JuliaDiffEq